tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+requires =
+    setuptools >= 30.3.0
+    pip >= 19.3.1
+isolated_build = true
+
+
+[testenv]
+extras = test
```

Comment:
I think this extras should just be under the ``testenv:test`` section

Reply:
Doe the `changedir` also go there? I'm a little unclear on what the difference is between `testenv` and `testenv:test`...I had assumed that that the `testenv` stuff was included with `testenv:*` so that it should include anything common to both.